canonical/ubuntu.com
# CVE search results filters

## Done

- Added filtering functionality

## Known bugs and future work

### Affected packages
Will be handled in separate PR.   

### Status filter
There is a bug when trying to filter with an uneven number of releases and status. This comes from the security API itself, it groups a single version and a single status together and throws 500 if it can't make a pair. The fix needs to be implemented on that codebase first and will be handled in a separate PR. 

### No results view
Currently not implemented, will be taken care of in separate ticket. 

### Show maintained releases link
This link (found under the releases filter) is currently not implemented. Waiting on a design to explain the correct functionality for this and will implement this in a separate PR.

## QA

- View the site locally in your web browser at: https://ubuntu-com-13494.demos.haus/security/cves
- Check that each release link routes to the search results with the correct filters applies
- Click the clear all filters button and see that it works as expected
- Play with the filters (please keep in mind the status filter bug) and see that there are no abnormalities. It might help to cross reference the total results for any given query against the live site. 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8029
